index.html

```diff
@@ -30,43 +30,43 @@ <h1 class="title">Step 1</h1>
                 </div>
             </div>
             <div class="field">
-                <label for="encryption">Encryption: <a target="_blank" rel="noopener" href="https://tools.ietf.org/html/draft-foudil-securitytxt-04#section-3.4.3">(description)</a></label>
+                <label for="encryption">Encryption: <a target="_blank" rel="noopener" href="https://tools.ietf.org/html/draft-foudil-securitytxt#section-3.5.4">(description)</a></label>
                 <div class="control">
                     <input name="encryption" id="encryption" class="input" placeholder="https://example.com/pgp-key.txt">
                 </div>
             </div>
             <div class="field">
-                <label for="acknowledgements">Acknowledgements: <a target="_blank" rel="noopener" href="https://tools.ietf.org/html/draft-foudil-securitytxt-04#section-3.4.1">(description)</a></label>
+                <label for="acknowledgments">Acknowledgments: <a target="_blank" rel="noopener" href="https://tools.ietf.org/html/draft-foudil-securitytxt#section-3.5.1">(description)</a></label>
                 <div class="control">
-                    <input name="acknowledgements" id="acknowledgements" class="input" placeholder="https://example.com/acknowledgements.html">
+                    <input name="acknowledgments" id="acknowledgments" class="input" placeholder="https://example.com/hall-of-fame.html">
                 </div>
             </div>
             <div class="field">
-                <label for="permission">Permission: <a target="_blank" rel="noopener" href="https://tools.ietf.org/html/draft-foudil-securitytxt-04#section-3.4.5">(description)</a></label>
+                <label for="preferredLanguages">Preferred-Languages: <a target="_blank" rel="noopener" href="https://tools.ietf.org/html/draft-foudil-securitytxt#section-3.5.7">(description)</a></label>
                 <div class="control">
-                    <input name="permission" id="permission" class="input" placeholder="none">
+                    <input name="preferredLanguages" id="preferredLanguages" class="input" placeholder="en, es, ru">
                 </div>
             </div>
             <div class="field">
-                <label for="policy">Policy: <a target="_blank" rel="noopener" href="https://tools.ietf.org/html/draft-foudil-securitytxt-04#section-3.4.6">(description)</a></label>
+                <label for="canonical">Canonical: <a target="_blank" rel="noopener" href="https://tools.ietf.org/html/draft-foudil-securitytxt#section-3.5.2">(description)</a></label>
                 <div class="control">
-                    <input name="policy" id="policy" class="input" placeholder="https://example.com/security-policy.html">
+                    <input name="canonical" id="canonical" class="input" placeholder="https://example.com/.well-known/security.txt.sig">
```

Comment:
This should say "https://example.com/.well-known/security.txt" without the ".sig" in the end

Reply:
Thanks for catching that @nightwatchcyber! I've gone ahead and fixed it.